code-challenges.test.js

```diff
@@ -13,39 +13,102 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersArray1 = [6, 7, 8, 9, 10]
-// Expected output: [18, 21, 24, 27, 30]
-const numbersArray2 = [24, 27, 30, 33, 36]
-// Expected output: [72, 81, 90, 99, 108]
+// const numbersArray1 = [6, 7, 8, 9, 10]
+// // Expected output: [18, 21, 24, 27, 30]
+// const numbersArray2 = [24, 27, 30, 33, 36]
+// // Expected output: [72, 81, 90, 99, 108]
 
 // b) Create the function that makes the test pass.
 
-// Pseudo code:
+
+// NOTE: I am having trouble running my codes on jest. I followed the instructions from the syllabus, but it keeps running test that is not specific to this week's assessment. I have the codes below, but I will need to seek help from one of the instructors regarding this.
```

Comment:
You have code written, but you don't have any tests created. Remember, jest tests use `describe`, `it`, and `expect` statements. Also, I would make sure you ran `yarn add jest`. If you didn't run that command in the terminal, you won't be able to run jest.